internal/command/run.go

```diff
@@ -105,6 +112,34 @@ func run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// Apply overrides from command line (optional)
+	//
+	for _, pstr := range overrideParams {
+		log.Print("Applying SCORE properties overrides...\n")
+
+		jsonBytes, err := json.Marshal(srcMap)
+		if err != nil {
+			return fmt.Errorf("marshalling score spec: %w", err)
+		}
+
+		pmap := strings.SplitN(pstr, "=", 2)
+		if len(pmap) <= 1 {
+			log.Printf("removing '%s'", pmap[0])
+			if jsonBytes, err = sjson.DeleteBytes(jsonBytes, pmap[0]); err != nil {
+				return fmt.Errorf("removing '%s': %w", pmap[0], err)
+			}
+		} else {
+			log.Printf("overriding '%s' = '%s'", pmap[0], pmap[1])
+			if jsonBytes, err = sjson.SetBytes(jsonBytes, pmap[0], pmap[1]); err != nil {
```

Comment:
How can numeric values be set? Looks like ony strings will be set here.

Suggest: Do what `humctl` does: the value needs to be a YAML value. i.e.
It will be a string UNLESS it is entirely a number. A number will be a sytring if quoted etc

e.g.:
```
abc -> string
123 -> number
"123" -> string
false -> boolean
'false' -> string

Reply:
Added support in https://github.com/score-spec/score-compose/pull/36